lib/CustomElements.js

```diff
@@ -7,7 +7,7 @@
  * Code distributed by Google as part of the polymer project is also
  * subject to an additional IP rights grant found at http://polymer.github.io/PATENTS.txt
  */
-// @version 0.7.20
+// @version 0.7.20-5f12843
```

Comment:
Yea looks that way. Did you grab the built tag? https://github.com/webcomponents/webcomponentsjs/tree/v0.7.21